.travis.yml

```diff
@@ -1,17 +1,18 @@
+dist: trusty
 language: scala
 scala:
-- 2.11.11
+- 2.12.2
 jdk:
-- oraclejdk8
+- openjdk8
 script:
-- sbt test
-before_deploy:
-  - pip install --user release-manager==0.3.0
+- sbt compile
 deploy:
+- provider: script
+  script: bash .travis/deploy.sh
   skip_cleanup: true
-  provider: script
-  script: ./.travis/deploy.sh $TRAVIS_TAG
   on:
-    # Deploy iff some tag is specified
     tags: true
-
+env:
+  global:
+  - secure: CInuuQEHT/apu/2t1JEI0TqifiGaUK/FwhYsNa3aUL+/fZQ84KKVlZIrD2FmAXuc8VRsM58SSRtR7MvZHqbAIPGDc3p0uojpVanRztIKfpxZCTve8qWKAhqh1BF1HfAXU1H7LAjCjEARfwtXctoNNtBx6OUxq64ssLpwqzydEbqTlApJy1AnrGYY+JuKu9SsZoLxe94A/+EgyczYnauWR8FxP8SDUb+r+DPeO1bAoU8bbpI4jzJH6IaJ4UAuybRPHjvOo+7cK/VGP6rpMD6/XwX0+61dnd8Sq9rQs2pLURA5GXYSewBgvfDCsUf0/iK9d3hEWkHBUs+KeOeNEM1E7s8dXvHEzzMRXrIvcG5BuMihXjMJb0XqKN0TBW6VYKc9KY+KmEH1ZQ80y8kvGFRUuYhwrFmcgPSSvm7s96pzsBkXVSvRfTGMzcXhc71RHtY7DGeGex2ibezV1cpXD88EPZKn+ksK8XUgH7mJO3iyQn/5+8gc6fuJvjPuchq11jn4nfwrlEzx56se8Uhg/5GuCGqXp0dyJ47AjmjIYmxLuCkyuYPRw2p9mAjk2DWDeqBoYtvPQ55RBfJusC6MFN33+S8uSvTiaYNLbQKSdlPW14Xkbcis2QJHVbwreHc/3VgIpSTM2hmrKIw1C/R4hO7FDvCH9HN+pZ2DlvhAimaNbgY=
```

Comment:
Thanks for adding suggestions with one clink button, but I can't just apply the change, as I have to rebase the branch to be 1:1:1 compliant.

Reply:
How do you know this one is username?

Reply:
You can rebase at the end, just before merging.
> How do you know this one is username?
I just assumed that we always start with username and then password, but that's just a guess indeed. Anyway would be good to know that these 2 lines are Docker related, in case many other env variables are added like [here](https://github.com/snowplow/snowplow/blob/master/.travis.yml#L19-L53)